src/progressbar/progressbar.spec.ts

```diff
@@ -0,0 +1,27 @@
+import {iit, it, ddescribe, describe, expect, injectAsync, TestComponentBuilder} from 'angular2/testing';
+
+import {Component} from 'angular2/angular2';
+
+import {NgbProgressbar} from "./progressbar";
+
+describe('ng-progressbar', () => {
+  describe('UI logic', () => {
```

Comment:
Could we add a bunch of tests for "business logic" - in the case of this directive we only care about percent calculation. I guess we should test a happy path and few corner cases